Library/Formula/simavr.rb

```diff
@@ -0,0 +1,24 @@
+class Simavr < Formula
+  desc "a lean, mean and hackable AVR simulator for linux & OSX "
+  homepage "https://github.com/buserror/simavr"
+  url "https://github.com/buserror/simavr/archive/v1.2.tar.gz"
+  sha256 "c7f645b323d74247409b50ea6915f1f324a7ee2a799ef107613626801defc070"
+
+  depends_on "osx-cross/avr/avr-gcc"
```

Comment:
You essentially build the dependency as a separate object rather than a dependency and if you need it at runtime install it into the formula's Cellar. Like an `.app` bundle to some extent.